src/molecules/Feature.tsx

```diff
@@ -0,0 +1,29 @@
+import type { FunctionComponent, ReactNode } from 'react';
+
+import type { FlexProps } from '../atoms';
+import { Box, Flex, Image } from '../atoms';
+
+export interface FeatureProps {
+  icon: string;
+  text: ReactNode;
+}
+
+export const Feature: FunctionComponent<FeatureProps & FlexProps> = ({ icon, text, ...props }) => (
```

Comment:
I wonder if these sort of components should be named differently. I.e. `LandingFeature` or whatever. So that we don't confuse components when building "actual app UIs"

Reply:
Thoughts?

Reply:
I wasn't sure what to name this one. It's used for the Quill downloads page too. Do you have any better suggestions?

Reply:
See Slack for discussion